lib/cli/src/commands/app/purge_cache.rs

```diff
@@ -0,0 +1,54 @@
+//! Get information about an edge app.
+
+use super::util::AppIdentOpts;
+use crate::{
+    commands::AsyncCliCommand,
+    opts::{ApiOpts, ItemFormatOpts},
+};
+
+/// Purge caches for applications.
+///
+/// Cache scopes that can be cleared:
+/// * InstaBoot startup snapshots
+///   Will delete all existing snapshots.
+///   New snapshots will be created automatically.
+#[derive(clap::Parser, Debug)]
+pub struct CmdAppPurgeCache {
+    #[clap(flatten)]
+    #[allow(missing_docs)]
```

Comment:
We should have the docs for this

Reply:
No we should not, because the documentation is on the nested field.

I recommend reading the comments on the struct.

Reply:
Will the CLI show those comments?